prow/config/jobs/istio.yaml

```
@@ -217,6 +217,15 @@ jobs:
     type: presubmit
     command: [make, gen-check]
 
+  - name: checkReleaseNotes
+    type: presubmit
+    command:
+      - ./tools/check_release_notes.sh
+      - --token-path=/etc/github-token/auth
+    requirements: [github]
+    modifiers:
+      - skipped
+
```

In order to have access to the `check_release_notes.sh` script in the job, you will need to also clone the `istio/test-infra` repo then update the script path: (e.g. `../test-infra/tools/check_release_notes.sh`) ([example](https://github.com/istio/test-infra/blob/2c406a26aa936c06220291dd77656b0ec44ac2ad/prow/config/jobs/istio.io.yaml#L29)). 

```suggestion
repos: [istio/test-infra@master]
```

---

Thanks.